galaxy/accounts/migrations/0007_auto_20180821_1259.py

```diff
@@ -0,0 +1,68 @@
+# -*- coding: utf-8 -*-
+# Generated by Django 1.11.15 on 2018-08-21 16:59
+from __future__ import unicode_literals
+
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('accounts', '0006_remove_customuser_github_user'),
+    ]
+
+    operations = [
+        migrations.RemoveField(
+            model_name='customuser',
+            name='cache_refreshed',
+        ),
+        migrations.RemoveField(
+            model_name='customuser',
+            name='karma',
+        ),
+        migrations.AddField(
```

Comment:
If this list is going to grow, it might be reasonable to use standalone table for notification preferences.

Reply:
Then it becomes a one-to-one relationship, which seams silly.